tests/Microsoft.Data.Analysis.Tests/DataFrameTests.cs

```diff
@@ -1418,6 +1464,16 @@ public void TestColumnFilter()
             Assert.Equal(7, filtered[3]);
         }
 
+        [Fact]
+        public void TestMeanAndMedian()
+        {
+            Int32DataFrameColumn ints = new Int32DataFrameColumn("Ints", new int?[] { 1, 3, 3, 2, null, 9, 10 });
```

Comment:
The "median" test here isn't very great because if you don't ignore the `null`, the median is still `3`. Can we get a different set that would actually change the median if the `null` wasn't considered?